vm/error.go

```diff
@@ -106,6 +110,8 @@ func (k ErrorKind) String() string {
 		return "invalid split commitment proof"
 	case ErrInvalidRootAsset:
 		return "invalid zero-value root asset"
+	case ErrUnFinalizedAsset:
+		return "un-finalized asset"
```

Comment:
ultra-nit: unfinalized is a correct word and doesn't need a hyphen.

Reply:
Hmm, both my IDE and translation website of choice (dict.leo.cc) didn't want to know that word. But you're right, reverted.